docs/Build-for-Edison-with-mraa.md

```diff
@@ -0,0 +1,79 @@
+# Build IoT.js with mraa for Intel Edison
+
+Edison board has Yocto Linux OS, so it is a enhanced case in [Build for Linux](https://github.com/Samsung/iotjs/wiki/Build-for-Linux).
+Edison board has mraa inside, so we can implement IoT.js's gpio module with mraa. See [#295](https://github.com/Samsung/iotjs/pull/295).
+
+## Overall steps
+
+1. Get the libmraa.so and related headers in Edison's Yocto OS, and put them into the host PC (Ubuntu 14.04)
+2. Build the iotjs with mraa in the host PC
+3. Copy the binary into the Edison Yocto OS.
+
+## Prerequisite
+
+* mkdir a folder in iotjs for storing mraa related files.
+
+```
+git clone https://github.com/Samsung/iotjs.git
+cd iotjs
+mkdir mraa
+```
+
+## 1. Get the mraa library and headers
+
+[This official page] step1, step3 and step4 (https://software.intel.com/en-us/get-started-edison-linux) describe how to get started with edison and how to setup its network. Then we can get Edison's IP.
```

Comment:
The markdown link format is broken here. Please remove the brackets or write like this:
`Step 1, 3 and 4 of [this official page](https://software.intel.com/en-us/get-started-edison-linux)  describe...`
It'll look like this:
Step 1, 3 and 4 of [this official page](https://software.intel.com/en-us/get-started-edison-linux)  describe...